openapi.yaml

```diff
@@ -0,0 +1,92 @@
+swagger: '2.0'
+info:
+  description: Endpoint for perfoming OAuth2 operations
+  title: OAuth2
+  version: v1
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+schemes:
+  - https
+host: api.oregonstate.edu
+produces:
+  - application/json
+consumes:
+  - application/x-www-form-urlencoded
+paths:
+  /oauth2/token:
+    post:
+      description: Get an access token to make API calls. Part of OAuth2 client credentials flow.
+      consumes:
+        - application/x-www-form-urlencoded
```

Comment:
I was going to write this to use a JSON body but it looks like it's not possible? I checked the [postman scripts](https://github.com/osu-mist/postman-tools/blob/master/authentication-pre-request.js#L19) as well and it looks they're using `application/x-www-form-urlencoded` too